<!DOCTYPE html>
<html lang="en">
<head>
	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-214102016-1"></script>
	<script>
  	window.dataLayer = window.dataLayer || [];
  	function gtag(){dataLayer.push(arguments);}
 	 gtag('js', new Date());

  	gtag('config', 'UA-214102016-1');
	</script>


  <title>Mutation blocks</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="author" content="Amna farooq">
  <meta name="keywords" content="methyaltion, mutation, differential methyaltion, integrative bioinformatics">
  <meta name="description" content="supplemtary data for regulatory mutation blocks paper follicular lymphoma 2021">
  <meta name="revised" content="25/11/2021">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  
  <style>
	//.head{
	  //background-color : white;
	  //padding: 40px;
	  //text-align: center;
	  //}

		.head{
	  background-color : white;
	  padding: 50px;
	  text-align: center;
	  font-family:helvatica;

	  //border-style: groove;
	  //border-radius: 25px;
	  //margin-bottom: 20px;
	  }
	  .author{
	  background-color : lightblue;
	  padding: 10px;
	  text-align: center;
	  font-family:helvatica;
	  //height : 100px;
	  
	  }
	  .author_des{
	  background-color : white;
	  padding: 20px;
	  text-align: center;
	  font-family:helvatica;
	  font-size:130%;
	  //height : 100px;
	  
	  }

	  .abstract{
	  background-color :lightblue;
	  padding: 30px;
	  font-family:helvatica;
	  font-size:110%;

	  }

	  .abstract_des{
	  background-color :white;
	  padding: 40px;
  	  font-family:helvatica;
  	  font-size:160%;
  	  border-style: groove;
	  border-radius: 25px;
	  margin: 50px;

	  }
	  
	  
		
	.supFiles{
	  background-color :lightblue;
	  padding: 30px;
	  	  font-family:helvatica;
	  	  font-size:160%;

	  }
	  .contets_des{
	  background-color :white;
	  padding: 60px;
	  	  font-family:helvatica;
	  	  font-size:160%;

	  }
	  
	.col_names{
	  background-color :lightyellow;
	  padding: 20px;
	  margin-left:50px;
	  margin-bottom: 50px;

		border:2px;
		border-style:solid;
		border-color:#808080;
	  }
	.ref{
	  background-color :lightblue;
	  padding: 30px;
	  	  font-family:helvatica;
	  	  font-size:160%;

	  }
	  .ref_des{
	  background-color :white;
	  padding: 60px;
	  	  font-family:helvatica;
	  	  font-size:160%;

	  }

	  .footer{
	  background-color :#4FEDB8;
	  padding: 10px
	  }
  </style>
</head>

<body>

<!-- Header -->

<div class = "container-fluid head">

	<div class="row", style="align: center"  > 
		<div class="col-sm-3">
			<img src="Radiumhospitalets-legater.png" class="img-thumbnail" alt="Radiumhospitalets legater" >
		
		</div>
		<div class="col-sm-3">
			<img src="helsesorost.png" class="img-thumbnail" alt="helsesorost" >
		</div>
		<div class="col-sm-3">
			<img src="ous.png" class="img-thumbnail" alt="Oslo University Hospital" >
		
		</div>
	</div>
	<hr style="border-top: dotted 3px grey  ;margin-top: 80px ; margin-bottom: 80px" />
	<div class="row"> 
		
			<h1>Integrating whole genome sequencing, methylation, gene expression and topological associated domain information in regulatory mutation prediction:  a study of follicular lymphoma. 
			</h1>
	
	<hr style="border-top: dotted 3px grey ; margin-bottom: 80px ;margin-top: 80px ;" />

	</div>
</div>


<!-- First Section header -->



<!-- Second Section -->
<div class = "container-fluid author">

	<div class="row"> 
		
			<h2> AUTHORS:</h2>
	
	</div>
</div>

<div class = "container-fluid author_des">

	<div class="row"> 
		
			<strong> Amna Farooq <sup>1</sup>, Gunhild Trøen <sup>1</sup>, Jan Delabie <sup>3</sup> and Junbai Wang <sup>2,4*</sup> </strong>
			<p></p>
			<p></p>
			<p>1.	Department of Pathology, Oslo University Hospital - Norwegian Radium Hospital, Oslo, Norway </p>
			
			<p>2.	Institute for Clinical Medicine, University of Oslo, Norway.</p>
				
			<p>3.	Laboratory Medicine Program, University Health Network and University of Toronto, Toronto, Ontario, Canada.</p>

			<p>4.	Department of clinical molecular biology (EpiGen), Akershus University Hospital, Lørenskog, Norway.</p>


			<p>*To whom correspondence should be addressed.Email: junbai.wang@medisin.uio.no </p>
	
	</div>
</div>


<!-- Third Section -->
		

<div class = "container-fluid abstract">

	<div class="row", style="text-align: center"> 
		
			<h2> ABSTRACT:</h2>
	
</div>

<div class = "container-fluid abstract_des">

	<div class="row"> 
		
			<p> A major challenge in human genetics is of the analysis of the interplay between genetic and epigenetic factors in a multifactorial disease like cancer. To understand this interplay, it is important to comprehensively analyze genetic and epigenetic features. Here, a novel methodology is proposed to investigate genome-wide regulatory mechanisms in cancer, as studied with the example of follicular Lymphoma (FL). In the first phase, a new machine-learning method is designed to identify Differentially Methylated Regions (DMRs) by computing six attributes. In the second phase, an integrative data analysis method is developed to study regulatory mutations in FL, by considering differential methylation information together with DNA sequence variation, differential gene expression, 3D organization of genome (e.g., topologically associated domains - TADs), and enriched biological pathways. Resulting mutation block-gene pairs are further ranked to find out the significant ones. By this approach, ~159 predicted mutation block-gene pairs with possible relevance to FL were identified. Notably, BCL2 and BCL6 were identified as top-ranking FL-related genes with several mutation blocks and DMRs acting on their regulatory regions. Two additional genes, CDCA and CTSO4, were also found in top rank with significant DNA sequence variation and differential methylation in neighboring areas, pointing towards their potential use as biomarkers for FL. This work provides a novel method for combining both genomic and epigenomic information to investigate genome-wide gene regulatory mechanisms in cancer and contribute to devising novel treatment strategies. </p>
	</div>
</div>

			

		
</div>

<!-- 1st Section  supFiles -->

<div class = "container-fluid supFiles">

	<div class="row", style="text-align: center"> 
		
			<h2> SUPPLEMENTARY MATERIAL:</h2>
	</div>
</div>

<div class = "container-fluid contets_des">

	<div class="row"> 
		
			<p>Following are the additional supplemtary files containing detailed results data. Data can be used to replicate the results reported in main study or can be used a resource to conduct futher research of Follicular lymphoma.</p>
			<ul>
				<li><a href="https://1drv.ms/u/s!ApvEfA3OSOa8hFyqp5-0SJC1TOc2?e=7jOAdf"> DMR_0.7.bed</a>
				 : File containing whole genome list of DMRs identified by our method . DMRs were filtered for rank score 0.7. File has following coloumns: </li>

				<div class = "container-fluid col_names">
				chr &emsp;start&emsp;end&emsp;mr_info(chr:mrID:mr_sites:Hyper_hypo_mix_dmr:type_of_DMR)&emsp;score(from 0 to 1)
				</div>
				<p></p>
				<p></p>

				<li><a href="https://amnfar.github.io/FL_project/data/mutation_blocks_summary.tsv"> mutation_blocks_summary.tsv</a>
				 : Detailed list of all mutation blocks identified by current study. File has following coloumns: </li>

				<div class = "container-fluid col_names">
				block_id&emsp;chrom&emsp;start_pos&emsp;end_pos&emsp;number_of_mutations&emsp;number_of_patients&emsp;mutation_distribution
				</div>
				<p></p>
				<p></p>

				<li><a href="https://amnfar.github.io/FL_project/data/DMR_mutation.tsv"> DMR_mutation.tsv</a>
				 : List of all DMRs overlapping to mutation blocks. File has following coloumns: </li>

				<div class = "container-fluid col_names">
				chrs&emsp;start_pos&emsp;end_pos&emsp;block_id&emsp;mr_chrs&emsp;mr_start_pos&emsp;mr_end_pos&emsp;mr_sites&emsp;mr_logReg_proba
				</div>
				<p></p>
				<p></p>

				<li><a href="https://amnfar.github.io/FL_project/data/differentially_expressed_genes_1.25Fd.txt"> differentially_expressed_genes_1.25Fd.txt</a>
				 : List of differentially expressed genes in FL (14 tutor vs 4 normal) filtered on 1.25 fd. Quantile normalized RPKM levels for each gene in corresponding sample are given.
					The 1.25fold diffExp genes are RPKM values calculated by gene length( Ref https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-021-02936-w). File has following coloumns: </li>

 				<div class = "container-fluid col_names">
				#gene&emsp;differential_expression_T_test_p_value&emsp;4170686/tumor&emsp;4177987_tumor&emsp;4160468_tumor&emsp;4121361_tumor&emsp;4159170_tumor&emsp;4134005_tumor&emsp;4139696_tumor&emsp;4177601_tumor&emsp;4105105_tumor&emsp;4175837_tumor&emsp;4158726_tumor&emsp;4189200_tumor&emsp;4174905_tumor&emsp;4188900_tumor&emsp;normal_gcb_cells_SRR834985&emsp;normal_gcb_cells_SRR834984&emsp;normal_gcb_cells_SRR834986&emsp;normal_gcb_cells_SRR834983
				</div>
				<p></p>
				<p></p>

				
				<li><a href="https://amnfar.github.io/FL_project/data/TAD_summary.tsv"> TAD_summary.tsv</a>
				 : List of all TADs overlapping to genes, mutation blocks and enhancers. TAD2gene = if gene exists in TAD, Boundary2gene = if gene exsists in TAD boundary, TAD2block= if mutation block exists in the TAD, Boundary2block = if the mutation block exists in the TAD boundary.   isTAD = will be one if the mutation block and gene exsist in same TAD . File has following coloumns: </li>
				<div class = "container-fluid col_names">
					gene_name&emsp;patients&emsp;gene_type&emsp;block_id&emsp;new_mr_sites&emsp;patient_id&emsp;enhancers&emsp;TAD2gene&emsp;Boundary2gene&emsp;TAD2block&emsp;Boundary2block&emsp;isTAD
				</div>
				<p></p>
				<p></p>
				


				<li ><a href="https://amnfar.github.io/FL_project/data/gene_summary.csv"> gene_summary.csv</a>
				 : Final 159 gene list after applying DAVID/GO pathway enrichment analysis (p-values <0.05) on the unique genes for mutation blocks with overlapping to either DMR or DEG regions and with feature score>=0.5. File has following coloumns: </li>
				
				<div class = "container-fluid col_names">
				gene_name&emsp;gene_type&emsp;block_id&emsp;new_mr_sites&emsp;patients&emsp;isTAD&emsp;enhancers&emsp;patient_id&emsp;geneType_meanScore&emsp;num_of_patients&emsp;diffExp_pval&emsp;dmr_pval&emsp;median_scores&emsp;mean_scores
				</div>
				<p></p>
				<p></p>	

				<li ><a href="https://amnfar.github.io/FL_project/data/GO_term_gene.csv"> GO_term_gene.csv</a>
				 : Final 159 gene list after applying DAVID/GO pathway enrichment analysis (p-values <0.05) on the unique genes for mutation blocks with overlapping to either DMR or DEG regions and with feature score>=0.5. File has following coloumns: </li>
				 <div class = "container-fluid col_names">
				 Category&emsp;Term&emsp;Count&emsp;%&emsp;PValue&emsp;Genes&emsp;List Total&emsp;Pop Hits&emsp;Pop Total&emsp;Fold Enrichment&emsp;Bonferroni&emsp;Benjamini&emsp;FDR&emsp;total_patients
				 </div>
				<p></p>
				<p></p>
					
				<li ><a href="https://amnfar.github.io/FL_project/data/s_blocks_summary.tsv"> s_blocks_summary.tsv</a>
				 : mutation blocks predicted by BayesPI-BAR2 tool in the DMRs overlapping with enhancer regions.(reference supplementary method and results). File has following coloumns: </li>
				 <div class = "container-fluid col_names">
				block_id&emsp;chrom&emsp;start_pos&emsp;end_pos&emsp;number_of_mutations&emsp;number_of_patients&emsp;mutation_distribution&emsp;region_names
				</div>
				<p></p>
				<p></p>
				
				<li ><a href="https://amnfar.github.io/FL_project/data/significant_blocks4mr_enhancer_regions.txt"> significant_blocks4mr_enhancer_regions.txt</a>
				 : Significant mutation blocks predicted by BayesPI-BAR2 tool in the DMRs overlapping with enhancer regions.(reference supplementary method and results). This file can be used to extract significant blocks from s_blocks_summary.tsv. File has one coloumn only contiating mutation block IDs. </li>
				
				<p></p>
				<p></p>
				<li ><a href="https://amnfar.github.io/FL_project/data/enh_target_gene.zip"> enh_target_gene.zip</a>
				 : Enhancer target genes for the enhnacers around four selected genes (BCL2,BCL6,CDCA4,CTSO). </li>
				<p></p>
				<p></p>
				
				<li ><a href="https://amnfar.github.io/FL_project/data/differentially_expressed_genes_exon.txt"> differentially_expressed_genes_exon.txt</a>
				 : Differentially expressed genes in FL (14 tutor vs 4 normal) with RPKM values normalized on exon length. File has following coloumns: </li>
				 <div class = "container-fluid col_names">
				#gene&emsp;differential_expression_T_test_p_value&emsp;4170686&emsp;4121361&emsp;4160468&emsp;4177987&emsp;4139696&emsp;4134005&emsp;4159170&emsp;4177601&emsp;4105105&emsp;4189200&emsp;4158726&emsp;4175837&emsp;4188900&emsp;4174905&emsp;SRR834983&emsp;SRR834986&emsp;SRR834984&emsp;SRR834985
				</div>
				<p></p>
				<p></p>
				
				<li ><a href="https://github.com/amnfar/FL_project/tree/main/data/Results%20of%20three%20tests%20(327%2C758%2C%20and%20959%20genes)%20on%207%20chromatin%20states"> Results of three tests (327,758, and 959 genes) on 7 chromatin states</a>
				 : Folder contains results for robustness analysis of mutation block-gene associations by evaluating seven chromatin states with chromatin state data gethered from Hoffman etal .</li>
				
				<p></p>
				<p></p>
			</ul>	
	</div>
</div>




<!-- 2nd Section  References -->

<div class = "container-fluid ref">

	<div class="row", style="text-align: center"> 
		
			<h2> REFERENCES:</h2>
	</div>
</div>

<div class = "container-fluid ref_des">

	<div class="row"> 
		
			<ol>
				<li> Batmanov, K., J. Delabie, and J. Wang, BayesPI-BAR2: A New Python Package for Predicting Functional Non-coding Mutations in Cancer Patient Cohorts. Front Genet, 2019. 10: p. 282.</li>
				<li> Batmanov, K., et al., Integrative whole-genome sequence analysis reveals roles of regulatory mutations in BCL6 and BCL2 in follicular lymphoma. Scientific Reports, 2017.</li>
				<li> Farooq, A., et al., HMST-Seq-Analyzer: A new python tool for differential methylation and hydroxymethylation analysis in various DNA methylation sequencing data. Computational and structural biotechnology journal, 2020. 18: p. 2877-2889.</li>

				<li> Kretzmer, H., et al., DNA methylome analysis in Burkitt and follicular lymphomas identifies differentially methylated regions linked to somatic mutation and transcriptional control. Nat Genet, 2015. 47(11): p. 1316-25.</li>
				<li> Hoffman, M.M., et al., Integrative annotation of chromatin elements from ENCODE data. Nucleic Acids Res, 2013. 41(2): p. 827-41. </li>
			</ol>
	</div>
</div>

</body>
</html>
